src/notifications/use-push-notifications.tsx

```diff
@@ -74,19 +92,35 @@ export const usePushNotifications = () => {
       return;
     }
     setStatus(permissionStatus);
-    return token;
   }, [register]);
 
-  return {
-    status,
-    config: configQuery.data,
-    updateConfig: configMutation.mutate,
-    checkPermissions,
-    requestPermissions,
-    register,
-  };
+  return (
+    <NotificationContext.Provider
+      value={{
+        status,
+        config: configQuery.data,
+        updateConfig: configMutation.mutate,
+        checkPermissions,
+        requestPermissions,
+        register,
+        fcmToken,
+      }}
+    >
+      {children}
+    </NotificationContext.Provider>
+  );
 };
 
+export function useNotificationContextState() {
```

Comment:
I can see that the naming of these context hooks is a bit inconsistent throughout the app. Some have only `Context` as postfix, some have only `State`, and some have no postfix. Personally I think I prefer the latter, i.e. just `useNotifications`.

Reply:
Agreed 👍